.ahub/tcchecker-tca/config.yaml

```diff
@@ -29,6 +22,7 @@ test:
         - functionName:
             starts:
               - TEST
+              - TYPED_TEST
```

Comment:
I found out some new tests use `TYPED_TEST` but it was not counted. 

By this PR:  266 (negative) / 495 (total)

@seanshpark 
I didn't update compiler configuration because some tests use `TYPED_TEST` for `luci-interpreter` but most tests introduced before counting negative test. (I checked commit date).

Reply:
(out of curiosity)
```
# of negative  = 266
# of positive = 495 -266 = 229
```
is it right? What is the requirement? # of negative >= # of positive?

Reply:
@hseok-oh (optional) where is the information(manual) about "TYPED_TEST" ?

Reply:
"TYPED_TEST" is gtest feature. (https://google.github.io/googletest/reference/testing.html#TYPED_TEST)

Reply:
>is it right? What is the requirement? # of negative >= # of positive?

Right.